Kickstarter-iOS/Views/Transitions/ScrollingTransitionAnimator.swift

```diff
@@ -0,0 +1,40 @@
+import UIKit
+
+internal final class ScrollingTransitionAnimator: NSObject, UIViewControllerAnimatedTransitioning {
+  weak var transitionContext: UIViewControllerContextTransitioning?
+  var tabBarController: UITabBarController!
```

Comment:
Is there any reason you need to hold the references to `tabBarController`, `lastIndex` and `transitionContext`?